lib/nitcorn/examples/src/nitcorn_reverse_proxy.nit

```diff
@@ -0,0 +1,9 @@
+import nitcorn
```

Comment:
The license is missing, and you should add some explanations within this example.